src/slices/LusdSlice.js

```diff
@@ -0,0 +1,67 @@
+import { ethers } from "ethers";
```

Comment:
Since this is a new file. do you mind using TS for this? :) intentional TypeScipt at the time of writing this is definitely better than retroactively going back and adding objects.

Reply:
Yes, let me work on that.

Reply:
resolved in https://github.com/OlympusDAO/olympus-frontend/pull/453/commits/f226a02fe86094d69c9d25890ab62bc383928afd